python/py-sphinx-bootstrap-theme/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        ryan-roemer sphinx-bootstrap-theme v0.7.1
+name                py-sphinx-bootstrap-theme
+platforms           darwin
+supported_archs     noarch
+license             MIT
+maintainers         nomaintainer
+
+description         Sphinx bootstrap theme.
+long_description    ${description}
+
+checksums \
+    rmd160  b57d6742d215223f9b4e995b3456ff00ae0c32ec \
+    sha256  df0cafc62db5768f2b595a0709099d2ff65bf19b1c19855bdfd9e1d6a1af6bfb \
+    size    1253883
+
+python.versions     27 36 37
+
+if {$subport ne $name} {
+    depends_build       port:py${python.version}-setuptools
+    livecheck.type      none
+}
+
+post-destroot {
```

Comment:
the ```post-destroot``` block should be within the ```if {$subport ne $name} {``` ; that will avoid the build errors Travis reports (same applies to the other port).